lib/json5.js

```diff
@@ -43,6 +43,7 @@ JSON5.parse = (function () {
             '\xA0',
             '\uFEFF'
         ],
+        operator = ['+', '-', '*', '/'],
```

Comment:
Support for arbitrary grouping using () would also round out this feature and make it more useful.